lib/proof/data/data_sources/witness_data_source.dart

```diff
@@ -30,9 +30,12 @@ class WitnessIsolatesWrapper {
 /// As this is running in a separate thread, we cannot inject [WitnessAuthLib]
 Future<Uint8List?> _computeWitnessAuth(WitnessParam param) async {
   final WitnessAuthV2Lib witnessAuthLib = WitnessAuthV2Lib();
-  final Uint8List? witnessBytes =
-      await witnessAuthLib.calculateWitnessAuth(param.wasm, param.json);
-  return witnessBytes;
+  return witnessAuthLib
+      .calculateWitnessAuth(param.wasm, param.json)
+      .catchError((error) {
+    print(error);
+    return null;
```

Comment:
This shadows the error, is it oki as we won't be able to differentiate a null or an error?

Reply:
removed